packages/frontend-plugin-api/src/wiring/createFrontendFeatureLoader.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ConfigApi } from '../apis/definitions';
+import { FrontendFeature } from './types';
+
+/** @public */
+export interface CreateFrontendFeatureLoaderOptions {
+  name: string;
```

Comment:
If possible I'd prefer to skip the name and use an approach similar to this instead: https://github.com/backstage/backstage/blob/45e7b21f665f9095691886d402cf8d4d90c3b3bf/packages/backend-plugin-api/src/wiring/createBackendFeatureLoader.ts#L51

Just looking to avoid requiring users to name things if it's not really needed x)